static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -427,10 +428,10 @@ class WidgetCardChart extends Component<WidgetCardChartProps, State> {
             );
           }
 
-          const shouldColorOther =
-            widget.displayType === 'top_n' &&
-            timeseriesResults &&
-            timeseriesResults.length > 5;
+          const otherRegex = new RegExp(`(?:.* : ${OTHER}$)|^${OTHER}$`);
+          const shouldColorOther = timeseriesResults
+            ?.map(series => series.seriesName)
+            .some(seriesName => seriesName.match(otherRegex));
```

Comment:
```suggestion
          const shouldColorOther = timeseriesResults?.some(series =>
            series.seriesName.match(otherRegex)
          );
```

Reply:
I went with:
```js
timeseriesResults?.some(series =>
  series.seriesName.match(otherRegex)
);
```

Thanks for the suggestion to shortening it!